distribution/docs/src/main/resources/content/_managing/_datamanagement/validating-data.adoc

```diff
@@ -0,0 +1,12 @@
+:title: Validating Data
+:type: dataManagementIntro
+:status: published
+:summary: Methods of correcting data validation errors.
+:order: 01
+
+== {title}
+
+Configure ${branding} to perform validation on ingested documents to verify the integrity of the metadata brought into the catalog.
+
+Isolate metacards with data validation issues and to edit the metacard to correct validation errors.
```

Comment:
`and to`? This reads like fragments of two different sentences.

Reply:
good catch.